sys/dev/pci/pci_user.c

```diff
@@ -133,6 +134,20 @@ struct pci_list_vpd_io64 {
  * We don't support PCIOCGETCONF_OLD64 because the earliest
  * COMPAT_FREEBSD64 architecture didn't exist until 9.0 (mips).
  */
+
+struct pci_bar_mmap64 {
+	uint64_t	pbm_map_base;	/* (void *) (sometimes IN)/OUT mmaped base */
+	size_t		pbm_map_length;	/* mapped length of the BAR, multiple
```

Comment:
Do we use size_t for other freebsd64 structs or should this be a uint64_t?

Reply:
Good point, `compat/freebsd64/freebsd64.h` uses `size_t` while freebsd32 does not. I am tempted to make it uint64_t anyway as it feels more reliable.

Reply:
I don't mind either way, but I've generally used size_t. Having `sizeof(size_t) != sizeof(uint64_t)` for a system that supported freebsd64 would be sufficiently catastrophic to the overall codebase that I'm not particularly worried about it.